src/components/activities/patientDetailsActivity/PatientDetailsActivity.tsx

```diff
@@ -440,13 +434,22 @@ const PatientDetailsActivity: FunctionComponent<TProps> = ({
                         content={PatientExams}
                       />
                     </div>
-                  ) : userSection == "admissions" ? (
-                    <div className="patientDetails__nested_content">
-                      <PatientDetailsContent
-                        title="Admissions"
-                        content={PatientAdmissions}
-                      />
-                    </div>
+                  ) : userSection === "visits" ? (
+                    patient?.data?.status === PatientDTOStatusEnum.O ? (
+                      <div className="patientDetails__nested_content">
+                        <PatientDetailsContent
+                          title="Visits"
```

Comment:
Title here should be OPD, right?

Reply:
Actually, the title is correct here.

Reply:
The title should be Visits both for inpatients and outpatients. But the content for outpatients will be the content of the OPD already implemented.